packages/graphql-extensions/src/index.ts

```diff
@@ -108,6 +112,16 @@ export class GraphQLExtensionStack<TContext = any> {
     );
   }
 
+  public didResolveOperation(o: {
+    requestContext: GraphQLRequestContext<TContext>;
+  }) {
+    this.extensions.forEach(extension => {
+      if (extension.didResolveOperation) {
+        extension.didResolveOperation(o);
```

Comment:
I guess we don't guard on whether or not the hook is a `typeof` `===` `'function'` in other places, so this isn't the exception to the pattern.